app/src/main/java/org/oppia/android/app/devoptions/DeveloperOptionsActivity.kt

```diff
@@ -15,7 +15,9 @@ const val EVENT_LOGS_FRAGMENT = "EVENT_LOGS_FRAGMENT"
 const val FORCE_NETWORK_TYPE_FRAGMENT = "FORCE_NETWORK_TYPE_FRAGMENT"
 
 /** Activity for Developer Options. */
-class DeveloperOptionsActivity : InjectableAppCompatActivity() {
+class DeveloperOptionsActivity :
+  InjectableAppCompatActivity(),
+  ForceCrashListener {
```

Comment:
Shift this to single like
`class DeveloperOptionsActivity : InjectableAppCompatActivity(), ForceCrashListener {`

Reply:
Done.